.gitignore

```diff
@@ -27,3 +27,5 @@ build/Release
 # Dependency directory
 # https://docs.npmjs.com/misc/faq#should-i-check-my-node-modules-folder-into-git
 node_modules
+.env
+.env.json
```

Comment:
This will ignore any files named `.env` and `.env.json` wherever they are which I think is fine. We also have files named `env.js`which we do NOT want to ignore. Is there a use case that I am missing?